dist/index.d.ts

```diff
@@ -1918,15 +1986,16 @@ interface GPUQueue
     size: GPUExtent3DStrict
   ): undefined;
   /**
-   * Schedules a copy operation of the contents of an image bitmap into the destination texture.
-   * @param source - {@link ImageBitmap} and origin to copy to `destination`.
+   * Issues a copy operation of the contents of a platform image/canvas
+   * into the destination texture.
+   * @param source - source image and origin to copy to `destination`.
    * @param destination - The texture subresource and origin to write to.
    * @param copySize - Extents of the content to write from `source` to `destination`.
    */
-  copyImageBitmapToTexture(
```

Comment:
We should keep this old version as deprecated, since the new version is not very available yet.